src/main/java/org/mitre/caasd/commons/CompositeCleaner.java

```diff
@@ -48,7 +48,7 @@ public CompositeCleaner(List<DataCleaner<? super T>> cleaners) {
      * @return A new CompositeCleaner
      */
     public static <T> CompositeCleaner<T> of(DataCleaner<? super T>... cleaners) {
-        return new CompositeCleaner<>(newArrayList(cleaners));
```

Comment:
You have to remove the <> ?

Reply:
As bizarre as this sounds -- yes.  

I double checked this too.  Compile in terminal, in IDE.  Both failed with the <>.  Both succeeded without the <>